ansible/vars/addresses/update.sh

```diff
@@ -16,5 +16,7 @@ function update_yaml() {
 for FILE in geth-*.holesky.yml;   do update_yaml "${FILE}"; done
 for FILE in erigon-*.holesky.yml; do update_yaml "${FILE}"; done
 for FILE in neth-*.holesky.yml;   do update_yaml "${FILE}"; done
-for FILE in linux-*.mainnet.yml;  do update_yaml "${FILE}"; done
+for FILE in nel-*.mainnet.yml;    do update_yaml "${FILE}"; done
+for FILE in erigon-*.mainnet.yml; do update_yaml "${FILE}"; done
+for FILE in geth-*.mainnet.yml;   do update_yaml "${FILE}"; done
```

Comment:
This change means that you'll have to generate new files in `ansible/vars/addresses` and remove old ones.